server/odc-core/src/main/resources/i18n/BusinessMessages_zh_CN.properties

```diff
@@ -680,6 +680,21 @@ com.oceanbase.odc.TaskType.GENERATE_ROLLBACK=备份回滚方案生成
 com.oceanbase.odc.TaskType.PRE_CHECK=预检查
 com.oceanbase.odc.TaskType.EXPORT_RESULT_SET=导出结果集
 com.oceanbase.odc.TaskType.APPLY_PROJECT_PERMISSION=申请项目权限
+com.oceanbase.odc.TaskType.SQL_PLAN=SQL 计划
+com.oceanbase.odc.TaskType.DATA_ARCHIVE=数据归档
+com.oceanbase.odc.TaskType.DATA_DELETE=数据清理
+
+#
+# TaskStatus
+#
+com.oceanbase.odc.TaskStatus.EXECUTION_SUCCEEDED=执行成功
```

Comment:
what does these for? what is the difference between this and `com.oceanbase.odc.core.shared.constant.TaskStatus`

Reply:
These properties are only for internationalization. The enums in `com.oceanbase.odc.core.shared.constant.TaskStatus` miss several states related to scheduling failure/timeout and approval.